src/sonar.ts

```diff
@@ -50,13 +84,26 @@ export function getSonarCloudInputs(): SonarCloudInputs {
   return { token, componentKey, apiUrl };
 }
 
-function buildSonarCloudUrl({
+function buildSonarCloudIssuesUrl({
   apiUrl,
   componentKey,
-}: SonarCloudInputs): string {
+}: SonarCloudInputs
+): string {
   const { prNumber } = getGitHubContext();
   const url = `${apiUrl}/issues/search?componentKeys=${encodeURIComponent(
     componentKey
-  )}&resolved=false`;
+  )}&resolved=false&ps=${MAX_PAGE_SIZE}`;
```

Comment:
Updating page size limit to 500 

Issue: https://github.com/pixee/upload-tool-results-action/issues/10